### ISSUE TITLE
C++ Exception: std::__1::system_error

### ISSUE THREAD

**Issue by JAStanton** [mattermost/mattermost-mobile#1625]
This is going to be really random, and a bit of a stretch for me. But my app has been throwing:
```
C++ Exception
std::__1::system_error
```
with no stack trace.

It's a React native app. My google-fu lead me to https://mattermost.atlassian.net/browse/MM-9080 which is similar. But it was resolved in private. I'm reaching out to you guys to see if you have any more context on the issue & if you managed to solve it. Was it application specific or something inherit w/ React Native? Any additional context may help.

Feel free to close this issue if you have no idea what I'm talking about :) thanks!

- Jonathan

**Comment by lindy65** [comment]
Thanks for the report @JAStanton 

Could you share which version of the mobile app you have installed?

Perhaps updating to the [latest version](https://about.mattermost.com/download/#mattermostApps) will resolve your issue?

If you are on the latest version, let me know and I'll check with our developers to see whether we've been seeing this in our app monitoring RN issues...

**Comment by lindy65** [comment]
@JAStanton is your issue resolved?

**Comment by JAStanton** [comment]
@lindy65 nope, let me rephrase:

I haven't used your app, but I noticed while building my own app that we shared the same crash: https://mattermost.atlassian.net/browse/MM-9080 but I don't have any access to see how your crash was resolved.

So out of curiosity I'm filing this request to see if you guys resolved it and if so what it was.

Obviously you don't have an SLA for helping people who aren't your customers but I was hoping it would be a quick lookup and maybe a link to a PR? :) if you want to close this ticket thats ok too, I was hoping it would be quick :P thanks!

- Jonathan

**Comment by lindy65** [comment]
Hi @JAStanton,

Thanks for your feedback,

The [ticket](https://mattermost.atlassian.net/browse/MM-9080) you referenced above was opened due to our automated app monitoring software finding the same crash as you are experiencing.  We couldn't reproduce the crash and, for now, we are monitoring for further occurrences of the same crash.

Have you perhaps taken a look at our documentation for [building your own app](https://docs.mattermost.com/mobile/mobile-compile-yourself.html)?  I'm not sure if there'll be anything useful in there...

**Comment by amyblais** [comment]
Hi @JAStanton, we haven't received an update so we'll assume that the problem is fixed or is no longer valid. 

If you still experience the same problem, try upgrading to the [latest version](https://about.mattermost.com/download/). 

If the issue persists, reopen this issue with the relevant information and we'd be glad to help you where we can.
